src/game.cpp

```diff
@@ -10872,24 +10872,45 @@ void game::chat()
     }
 
     uimenu nmenu;
-    nmenu.text = std::string( _("Who do you want to talk to?") );
-
+    nmenu.text = std::string( _( "Who do you want to talk to or yell?" ) );
```

Comment:
Should be: "Who do you want to talk to or yell at?"

Otherwise, this reads (without the first clause): "Who do you want to yell?"